e2e/cypress/integration/ui/settings/01_user_mgmt.spec.ts

```diff
@@ -26,81 +27,77 @@ describe('user management', () => {
     cy.get('app-user-table chef-button').contains('Create User').click();
     cy.get('app-user-management chef-modal').should('exist');
 
+    // adding a delay to more closely approximate an average human's type speed (200 char/min)
+    // the `should` will wait until the actual input value matches the expected value
+    // ref: https://github.com/cypress-io/cypress/issues/534
     cy.get('[formcontrolname=fullname]')
-      .type(name);
+      .type(name, { delay: 50 }).should('have.value', name);
 
     cy.get('[formcontrolname=username]')
-      .type(username);
+      .type(username, { delay: 50 }).should('have.value', username);
 
     cy.get('[formcontrolname=password]')
-      .type('chefautomate');
+      .type(password, { delay: 50 }).should('have.value', password);
 
     cy.get('[formcontrolname=confirmPassword]')
-      .type('chefautomate').then(() => {
-        // save new user
-        cy.get('[data-cy=save-user]').click().then(() => {
-          // confirm modal closed
-          cy.get('app-user-management chef-modal').should('not.be.visible');
-          // success alert displays
-          cy.get('chef-notification.info').should('be.visible');
-          // confirm new user row added
-          cy.contains(username).should('exist');
-          cy.contains(name).should('exist');
-        });
-      });
+      .type(password, { delay: 50 }).should('have.value', password);
+
+    // save new user
+    cy.get('[data-cy=save-user]').click();
+    cy.get('app-user-management chef-modal').should('not.be.visible');
+    cy.get('chef-notification.info').should('be.visible');
+
+    cy.get('chef-td').contains(username).should('exist');
+    cy.get('chef-td').contains(name).should('exist');
+  });
+
+  it('can view and edit user details', () => {
+    cy.route('PUT', '**/users/**').as('updateUser');
```

Comment:
Seems a very vague specification; can we not be more specific in the path...? If not, probably should have an in-line comment explaining why not. (Same thing for DELETE lower in this file.)

Reply:
I think the verb is specific enough in these cases.